pkgs/applications/misc/buku/default.nix

```diff
@@ -12,20 +11,42 @@ with pythonPackages; buildPythonApplication rec {
     sha256 = "1a33x3197vi5s8rq5fvhy021jdlsc8ww8zc4kysss6r9mvdlk7ax";
   };
 
+  nativeBuildInputs = [
+    pytestcov
+    pytest-catchlog
+    hypothesis
+    pytest
+  ];
+
   propagatedBuildInputs = [
     cryptography
     beautifulsoup4
     requests
     urllib3
   ];
 
-  phases = [ "unpackPhase" "installPhase" "fixupPhase" ];
+  preCheck = ''
+    # Fixes two tests for wrong encoding
+    export PYTHONIOENCODING=utf-8
+
+    ### Remove this for 3.1 ###
+    # See https://github.com/jarun/Buku/pull/167 (merged)
```

Comment:
https://github.com/jarun/Buku/pull/167 is merged, I added a note for the next version bump :)